tests/core/val-type/test_valtype_validate_argument.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+import pytest
+
+from wasm import (
+    constants,
+)
+from wasm.datatypes import (
+    ValType,
+)
+from wasm.exceptions import (
+    ValidationError,
+)
+
+
+@pytest.mark.parametrize(
```

Comment:
I've begun to think this is a bit of an anti-pattern.  If a parametrized test is testing two different classes of data (valid vs. invalid in this case), I think it's better to split them explicitly into two different parametrized tests instead of doing logic in the test function to distinguish them and requiring the inclusion of some marker value in the parameter tuple.